scripts/source-maintenance.sh

```diff
@@ -630,6 +771,13 @@ run_ collectAuthors || exit 1
 # Run formatting
 srcFormat || exit 1
 
-rm -f boilerplate_fix.sed
+if [ -z "$OnlyChangedSince" ]; then
+    sort -u <doc/debug-sections.tmp | sort -n >doc/debug-sections.tmp2
+    cat scripts/boilerplate.h doc/debug-sections.tmp2 >doc/debug-sections.txt
+    rm doc/debug-sections.tmp doc/debug-sections.tmp2
+else
+    echo "--only-changed-since specified, Skipping update of doc/debug-sections.txt"
+fi
```

Comment:
Something went wrong here. This PR should not be re-adding this existing code. Most likely, this `if` statement is not needed at all.

We just need to be careful about `boilerplate_fix.sed` removal (i.e. keep the line _after_ this `if` statement).

```suggestion
```